widen the dep in package:shelf_web_socket (#2421)
dart-lang/test: pkgs/test/CHANGELOG.md

## 1.25.11

> * Update the `package:shelf_web_socket` constraint to allow version `3.x`.

Reviewer: ```suggestion
* Update to be forward compatible with `package:shelf_web_socket` version `3.x`.
```
